tests/sentry/workflow_engine/endpoints/test_organization_detector_index.py

```diff
@@ -623,15 +623,17 @@ def setUp(self) -> None:
             "name": "Test Detector",
             "type": MetricIssue.slug,
             "projectId": self.project.id,
-            "dataSource": {
-                "queryType": SnubaQuery.Type.ERROR.value,
-                "dataset": Dataset.Events.name.lower(),
-                "query": "test query",
-                "aggregate": "count()",
-                "timeWindow": 3600,
-                "environment": self.environment.name,
-                "eventTypes": [SnubaQueryEventType.EventType.ERROR.name.lower()],
-            },
+            "dataSources": [
+                {
```

Comment:
this still won't be _quite_ right for the [format. see](https://getsentry.atlassian.net/browse/ACI-362).
```
dataSources: [
    {
        "type": "snuba_query_subscription"
        "queryObj": {
            "dataset": "events",
            "query": "",
            "aggregate": "count(transaction.duration)",
            "timeWindow": 3600,
            "environment": null,
        }
    }
]
```